Arduino/slave_node_example/slave_node_example.ino

```diff
@@ -0,0 +1,74 @@
+/*
+Example of a slave node using the CanOpen library based on Canfestivino
+Attach a potentiometer to A0
```

Comment:
Hmmm maar hier heb je over een potentiometer en een servo terwijl de EDS file gebruikt maakt van Sensor en Actuator dus het is inconsistent. 

Verander dit in de EDS file naar potentiometer en potmeter. 
Of hier naar Sensor en Actuator.

Reply:
Dat kan, het probleem is alleen bij de servo is dat je specifiek van een servo library maakt. Maar ik kan het wel `actuator` noemen ipv `myservo`. ook `POT_PIN` kan ik `SENSOR_PIN` noemen voor de consistentie?

Reply:
Ik zou gewoon Sensor en Actuator vervangen dan met Potentiometer en Servomotor (aangezien je gebruik maakt van de specifieke servo library). Daarbij geef je ook aan dat dit een voorbeeld is. Inprincipe zou het voor een developer niet veel uitmaken of het een Potentiometer is of een Generieke Sensor. Het gaat uiteindelijk om de implementatie en die blijft hetzelfde.

Reply:
Je kan natuurlijk ook die servo library eruit halen en dan nog wat simpler maken als je Servo en actuator wilt behouden